source/common/json/config_schemas.cc

```diff
@@ -73,6 +73,82 @@ const std::string Json::Schema::CLIENT_SSL_NETWORK_FILTER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::HTTP_CONN_NETWORK_FILTER_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "definitions" : {
+      "access_log": {
+        "type" : "object",
+        "properties" : {
+          "path" : {"type" : "string"},
+          "format" : {"type" : "string"},
+          "filter" : {"type" : "object"}
+        },
+        "required" : ["path"],
+        "additionalProperties" : false
+      },
+      "tracing" : {
+        "type" : "object",
+        "properties" : {
+          "operation_name" : {"type" : "string"},
+          "type" : {
```

Comment:
i think i cleaned that up, there should not be "type" at all under "tracing". Where did you see that?